src/z3c/dependencychecker/dependencychecker.py

```diff
@@ -343,6 +394,39 @@ def includes_from_zcml(path):
     return modules, test_modules
 
 
+def includes_from_plone_fti(path):
+    modules = []
+    test_modules = []
+    for path, dirs, files in os.walk(path):
+        for fti in [os.path.abspath(os.path.join(path, filename))
+                    for filename in files
+                    if fnmatch.fnmatch(filename, '*.xml')]:
+            contents = open(fti).read()
+            # check that we are on a Dexterity FTI and
+            # not any other random XML file
+            if contents.find('meta_type="Dexterity FTI"') == -1:
+                continue
+
+            found = [module for module in
+                     re.findall(FTI_BEHAVIOR_PATTERN, contents)
+                     if not module.startswith('.') and module.find('.') != -1]
+
+            found += [module for module in
+                      re.findall(FTI_SCHEMA_PATTERN, contents)
+                      if not module.startswith('.')]
+            found += [module for module in
+                      re.findall(FTI_KLASS_PATTERN, contents)
+                      if not module.startswith('.')]
+            found += [module for module in
+                      re.findall(FTI_PERMISSION_PATTERN, contents)
```

Comment:
Permissions are not real dotted names, it is just the name of the `IPermission` utility and it could be any string.
For example the permission [`plone.app.contenttypes.addDocument`](https://github.com/plone/plone.app.contenttypes/blob/master/plone/app/contenttypes/profiles/default/types/Document.xml#L20) cannot be imported, since it is just a name.
What is the idea behind including this?